token/rustfmt.toml

```diff
@@ -0,0 +1,10 @@
+comment_width = 80
+edition = "2021"
+format_code_in_doc_comments = true
+format_strings = true
+group_imports = "One"
+ignore = [
+  "**/*/entrypoint.rs"
+]
```

Comment:
Why is this here? We should definitely format entrypoints too!

Reply:
Because `rustfmt` will turn this:

```rust
use solana_program::{entrypoint, entrypoint::ProgramResult};
```

into this, which doesn't import the `entrypoint!` macro:

```rust
use solana_program::entrypoint::{self, ProgramResult};
```

I tried scouring the `rustfmt` docs for how to disable this import syntax alone, but didn't seem to find anything! But I agree, for the repo it should probably not be this general.

Reply:
bah that stinks. How about we change everywhere to call the macro with `solana_program::entrypoint!(process_instruction)`?

Reply:
This is the way